src/containerapp/azext_containerapp/containerapp_decorator.py

```diff
@@ -1012,9 +1013,19 @@ def construct_payload(self):
         self.set_up_service_bindings()
         self.set_up_unbind_service_bindings()
         self.set_up_source()
+        self.set_up_containers_for_private_registry_image()
         if self.get_argument_max_inactive_revisions() is not None:
             safe_set(self.new_containerapp, "properties", "configuration", "maxInactiveRevisions", value=self.get_argument_max_inactive_revisions())
 
+    def set_up_containers_for_private_registry_image(self):
```

Comment:
Nit: please consider renaming "private registry" to "default registry" as it's the wording we're pushing for now (and it is what is used in the UI)

Reply:
Thanks Dan! This change is not valid anymore in my latest commits.